radar/2023-09-12/grafana.md

```diff
@@ -5,4 +5,4 @@ quadrant:   tools
 tags:       [monitoring, alerting]
 ---
 
-Being evaluated for CHT Watchdog.
\ No newline at end of file
+Grafana is being evaluated for CHT Watchdog.
```

Comment:
I think we can promote this from "assess" to "adopt" now. @mrjones-plip What do you think?

Reply:
yup yup - we're all in!

Reply:
There are two blips on Grafana. This one where you commented is the initial version when we created the radar. The second blip can be found [here](https://github.com/medic/cht-tech-radar-contributors/blob/main/radar/2023-09-13/grafana.md), and it contains the updated status (adopt), plus extra info on Grafana. The radar merged the info from the two blips, and shows the newest one first. Let me know what you think.

Reply:
Of course. That's not obvious in the PR review UI  :/

Thanks for explaining.